plugins/module_utils/ilo_redfish_utils.py

```diff
@@ -51,6 +99,423 @@ def get_ilo_sessions(self):
         result["ret"] = True
         return result
 
+    def get_systems_data(self):
+        response = self.get_request(self.root_uri + self.systems_uri)
+        if not response["ret"]:
+            return response
+
+        return {
+            "ret": True,
+            "msg": response
+        }
+
+    def get_managers_data(self):
```

Comment:
Getting manager info is standard Redfish functionality; we have this today as "get_manager_inventory". Why is this needed in an iLO-specific module?

Reply:
I would much rather enhance "get_manager_inventory" in the standard module (or even have a different method in the standard module) to retrieve all properties in a desired manager. This is not something iLO-specific.

Reply:
Hi, Please refer the above comment, the same is applicable here.

Reply:
Resolving this conversation, thanks!